clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/TransferArraysV1Metadata.scala

```diff
@@ -0,0 +1,151 @@
+package org.broadinstitute.clio.transfer.model.arrays
+
+import java.net.URI
+import java.time.OffsetDateTime
+
+import org.broadinstitute.clio.transfer.model.TransferMetadata
+import org.broadinstitute.clio.util.model.DocumentStatus
+
+case class TransferArraysV1Metadata(
+  autocallCallRate: Option[Double] = None,
+  autocallDate: Option[OffsetDateTime] = None,
+  autocallGender: Option[Symbol] = None,
+  autocallPf: Option[Boolean] = None,
+  autocallVersion: Option[Int] = None,
+  beadPoolManifestFile: Option[URI] = None,
+  callRate: Option[Double] = None,
+  callRateThreshold: Option[Float] = None,
+  chipType: Option[String] = None,
+  clusterFile: Option[URI] = None,
+  controlSampleName: Option[String] = None,
+  createdAt: Option[OffsetDateTime] = None,
```

Comment:
These are timestamps in the database for when the row was created.  Don't we have this in clio

Reply:
Is the idea here to preserve the creation time from arrays database row? Not sure what the use of this field is. Anyway, elasticsearch itself doesn't automatically create a 'created at' property for you, and I don't think clio has this concept either.

Reply:
Aside from mistakes: the fields all come from some Arrays data source I consulted. CREATED_AT is a column in the ARRAYS_QC table in the cloud metrics database. (I'll drop the fields derived from the *_AT entries unless someone objects.)